WalletWasabi.Gui/Program.cs

```diff
@@ -43,7 +45,15 @@ private static void Main(string[] args)
 				AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
 				TaskScheduler.UnobservedTaskException += TaskScheduler_UnobservedTaskException;
 
-				runGui = ShouldRunGui(args);
+				try
+				{
+					runGui = ShouldRunGui(args);
```

Comment:
The method should probably be renamed as it does not indicate that it can run CLI daemon. Maybe `ProcessCliCommands`? Or something like that?

Reply:
ACK